spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source_spec.rb

```diff
@@ -1,10 +1,177 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource do
   let(:manager) do
-    FactoryBot.create(:provider_embedded_ansible, :with_authentication, :default_organization => 1).managers.first
+    FactoryBot.create(:provider_embedded_ansible, :default_organization => 1).managers.first
   end
+
+  # TODO:  Create a local repo instead... this will probably fail sporatically
+  # using a live repo
```

Comment:
My git foo late last night was not up to the task of figuring out how to do this, so I didn't bother...

But my guess is that this would be a better approach going forward.

Reply:
Lol I didn't see this before the meeting ... carry on.... :man_facepalming:

Reply:
Thinking I am going to fix this in a follow, for what it is worth.